lua/core/clock.lua

```diff
@@ -88,6 +94,7 @@ clock.cleanup = function()
 
   clock.transport.start = nil
   clock.transport.stop = nil
+  function clock.tempo_change_handler(bpm,source) end
```

Comment:
for consistency with clock.transport callbacks cleared here this could be set to `nil`, given that the check below will only call tempo_change_handler if it's not nil anyway.

```suggestion
  clock.tempo_change_handler = nil
```